src/usage/vector-type.md

```diff
@@ -0,0 +1,44 @@
+# Vector Types
+
+We have seen `vector` type in the previous section. In this section, we will show other vector types.
+
+## `bvector` binary vector
+
+The `bvector` type is a binary vector type in pgvecto.rs. It represents a binary vector, which is a vector where each component can take on two possible values, typically 0 and 1. 
+
+Here's an example of creating a table with a bvector column and inserting values:
+
+```sql {3}
+CREATE TABLE items (
+  id bigserial PRIMARY KEY,
+  embedding bvector(3) NOT NULL
+);
+
+INSERT INTO items (embedding) VALUES ('[1,0,1]'), ('[0,1,0]');
+```
+
+We support three operators to calculate the distance between two `bvector` values.
+
+- `<->` (`bvector_l2_ops`): squared Euclidean distance, defined as $\Sigma (x_i - y_i) ^ 2$.
+- `<#>` (`bvector_dot_ops`): negative dot product, defined as $- \Sigma x_iy_i$.
+- `<=>` (`bvector_cos_ops`): cosine distance, defined as $1 - \frac{\Sigma x_iy_i}{\sqrt{\Sigma x_i^2 \Sigma y_i^2}}$.
```

Comment:
We also support `jaccard` distance `<~>`, which defined as $1 - \frac{|X\cap Y|}{|X\cup Y|}$. And we can mention that `hamming` distance is the same with squared Euclidean distance.